ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ClusterSelection.tsx

```diff
@@ -1,9 +1,162 @@
+        clusters,
+        clusterIsPreSelected
+    );
+
+    useEffect(() => {
+        const clusterIds = clusters.map((cluster) => cluster.id);
+        const selectedClusterIds = clusterIds.filter((_, index) => selected[index]);
+        if (!isEqual(selectedClusterIds, formikValues.clusters)) {
+            setFieldValue('clusters', selectedClusterIds);
+        }
+    }, [selected, formikValues.clusters, setFieldValue, clusters]);
+
+    function renderTableContent() {
+        return clusters?.map(({ id, name }, rowIndex) => (
+            <Tr key={id}>
+                <Td
+                    key={id}
+                    select={{
+                        rowIndex,
+                        onSelect,
+                        isSelected: selected[rowIndex],
+                    }}
+                />
+                <Td>{name}</Td>
+            </Tr>
+        ));
+    }
+
+    function renderLoadingContent() {
+        return (
+            <Tr>
+                <Td colSpan={8}>
+                    <Bullseye>
+                        <Spinner isSVG />
+                    </Bullseye>
+                </Td>
+            </Tr>
+        );
+    }
+
+    function renderEmptyContent() {
+        return (
+            <Tr>
+                <Td colSpan={6}>
+                    <Bullseye>
+                        <EmptyStateTemplate
+                            title="No clusters found"
+                            headingLevel="h2"
+                            icon={SearchIcon}
+                        >
+                            {hasWriteAccessForCluster && (
+                                <Flex direction={{ default: 'column' }}>
+                                    <FlexItem>
+                                        <Text>Install a cluster to get started</Text>
+                                    </FlexItem>
+                                    <FlexItem>
+                                        <InstallClustersButton />
+                                    </FlexItem>
+                                </Flex>
+                            )}
+                        </EmptyStateTemplate>
+                    </Bullseye>
+                </Td>
+            </Tr>
+        );
+    }
+
+    function renderTableBodyContent() {
+        if (isFetchingClusters) {
+            return renderLoadingContent();
+        }
+        if (clusters && clusters.length > 0) {
+            return renderTableContent();
+        }
+        if (clusters && clusters.length === 0) {
+            return renderEmptyContent();
+        }
+        return null;
+    }
 
-function ClusterSelection(): ReactElement {
     return (
         <>
-            <div>Cluster selection</div>
+            <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                <Flex direction={{ default: 'column' }} className="pf-u-py-lg pf-u-px-lg">
+                    <FlexItem>
+                        <Title headingLevel="h1">Clusters</Title>
```

Comment:
Can you double check heading levels?
1. Wizard page renders **Create scan schedule** as `h1` element?
2. If yes, then **Clusters** as `h2` here?
3. If yes, then **No clusters**  as `h3` in empty state? By the way, **found** elsewhere implies no whatevers found according to **search filter** versus without found for no whatevers exist.

Reply:
Good callout, updated in [#8847](https://github.com/stackrox/stackrox/pull/8847)